xapian-bindings/go/Makefile.am

```diff
@@ -0,0 +1,38 @@
+GOSRCDIR = ${GOROOT}/src/xapian
+TESTS = ./test
+AM_TESTS_ENVIRONMENT = \
+	abs_builddir='$(abs_builddir)' ;\
+	srcdir='$(srcdir)'
+	export abs_builddir ;\
+	export srcdir ;
+LOG_COMPILER = ${GO} test ./test
+
+install-data-hook:
+	cd ${GOROOT}/src/xapian
+	${GO} install -i -x -a
```

Comment:
Each line in a make rule is run is a separate shell, so your `cd` has no effect on your `go install` command.